server/tests/src/test/java/org/infinispan/server/functional/rest/RestMetricsWithoutGaugeResourceIT.java

```diff
@@ -0,0 +1,38 @@
+package org.infinispan.server.functional.rest;
+
+import static org.infinispan.server.functional.rest.RestMetricsResourceIT.findMetric;
+import static org.infinispan.server.functional.rest.RestMetricsResourceIT.getMetrics;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.infinispan.server.test.core.ServerRunMode;
+import org.infinispan.server.test.junit5.InfinispanServerExtension;
+import org.infinispan.server.test.junit5.InfinispanServerExtensionBuilder;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+public class RestMetricsWithoutGaugeResourceIT {
+
+   private static final int NUM_SERVERS = 3;
```

Comment:
Why do we need 3 servers?

Reply:
Agreed: if you need clustered, 2 is enough.